torch/csrc/jit/tensorexpr/cuda_codegen.cpp

```diff
@@ -303,7 +309,7 @@ void CudaPrinter::visit(IntrinsicsPtr v) {
 }
 
 void CudaPrinter::visit(ExternalCallPtr v) {
-  throw unimplemented_lowering(v);
+  throw unimplemented_lowering(buildErrorMessage(std::to_string(v)));
```

Comment:
Why not have the `unimplemented_lowering` constructor call `buildErrorMessage` instead of doing it at every call site?